config/config_helpers.go

```diff
@@ -39,7 +39,6 @@ var TERRAFORM_COMMANDS_NEED_VARS = []string{
 	"plan",
 	"push",
 	"refresh",
-	"validate",
```

Comment:
Unintentional change?

Reply:
That's right. I synced original repo to my fork and it had a commit: 
https://github.com/gruntwork-io/terragrunt/commit/4f7fcb6d1ca0914589e0145e7d53f0721ac1ff5e

I rebase my branch and I ignored this file.